includes/connect.php

```diff
@@ -70,6 +70,13 @@
 $webhook_processPageDescription = $config["app_settings"]["webhook"]["processPageDescription"];
 
 
+function clean_data($data) {
+    $data = htmlspecialchars($data);
```

Comment:
throw in a strip_tags https://www.php.net/manual/en/function.strip-tags.php and you're golden